canvas/cli/common.py

```diff
@@ -1,16 +1,63 @@
+import json
 import sys
 
+DEFAULT_JSON = False
+DEFAULT_TABLE = False
+DEFAULT_SKIP_HEADERS = False
+
 # keys: [(items key, title, pretty title), ...]
-def cli_list(items, keys, table = False, skip_headers = False,
-        collective_name = 'items', sort_key = 'id'):
+def cli_list(items, keys, collective_name = 'items', sort_key = 'id', output_json = DEFAULT_JSON,
```

Comment:
I think we should update `table` to match `output_json`, so `output_table`.
Note that this requires updating the `dest` for `--table`.

(Note that the opposite is possible (using `json`), but then that conflicts with the `json` import.)